src/client/package.json

```diff
@@ -54,6 +54,12 @@
       "git add"
     ]
   },
+  "prettier": {
+    "semi": false,
+    "singleQuote": true,
+    "printWidth": 120,
+    "trailingComma": "all"
+  },
```

Comment:
Why does the prettier config also ned to be here?

Reply:
Paranoia involving VSCode — I'll remove.